internal/scripts/publish-vscode-extension.ts

```diff
@@ -80,7 +80,6 @@ async function packageAndPublish(version: string) {
 	switch (env.TLDRAW_ENV) {
 		case 'production':
 			await exec('yarn', ['package'], { pwd: EXTENSION_DIR })
-			await copyExtensionToReleaseFolder(version)
```

Comment:
Don't do it for production as it would retrigger a deploy. Figure out how to do it correctly.